arb_os/output.mini

```diff
@@ -343,14 +344,61 @@ public impure func queueMessageForSend(msg: ByteArray) {
     evmCallStack_queueMessage(msg);
 }
 
-public impure func sendQueuedMessages(queue: Queue) {
-    loop {
+public impure func sendQueuedMessages(queue: Queue, evmLogs: EvmLogs) -> EvmLogs {
+    return xloop {
```

Comment:
I don't love the fact that we're using an O(n^2) algorithm loop for this. If this get's charged to overhead, that could potentially be an issue

Reply:
Good catch. Switched to an O(n) algorithm.